ccm_web/client/src/components/MultipleSectionEnrollmentWorkflow.tsx

```diff
@@ -315,7 +315,7 @@ export default function MultipleSectionEnrollmentWorkflow (props: MultipleSectio
         return renderUpload()
       case CSVWorkflowState.Review:
         if (addEnrollmentsError !== undefined) {
-          return <BulkApiErrorContent error={addEnrollmentsError} file={file} tryAgain={resetUpload} />
+          return <BulkApiErrorContent error={addEnrollmentsError} file={file} tryAgain={resetUpload} context='enrolling a user' />
```

Comment:
I don't see any side-effect occurring expanding `APIErrorsTable ` to enroll user and create section feature